### PR TITLE
Add darts exercise

### PR DESCRIPTION
**Pull request by BNAndras**
Related to #236

### CODE REVIEW

File: exercises/practice/darts/darts.coffee [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Darts
+  score: (x, y) ->
```
Comment (ErikSchierboom) [inline exercises/practice/darts/darts.coffee:2]: Could this be a class method?

```suggestion
class Darts
  @score: (x, y) ->
```

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/practice/darts/darts.coffee
Comment (ladokp) [inline exercises/practice/darts/darts.coffee:1]: It looks like the tests are not synchronized with this change.

Comment (BNAndras) [reply exercises/practice/darts/darts.coffee:1]: Can you clarify please? It seems the tests.toml is up-to-date.
```
./bin/configlet sync --tests -e darts
Updating cached 'problem-specifications' data...
Checking exercises...
The `darts` exercise has up-to-date tests!
```

Comment (ladokp) [reply exercises/practice/darts/darts.coffee:1]: The initial code:
```
class Darts
  @score: (x, y) ->

module.exports = Darts
```

produces the following error:
`TypeError: darts.score is not a function`

Remove the @ in the solution and the error is gone.

**Pull request merged**